students/nartowska_karina/lesson_06_dicts_tuples_sets_args_kwargs/the_number_of_distinct_words_in_some_text.py

```diff
@@ -0,0 +1,7 @@
+def main():
```

Comment:
This code working properly for one line text. According to this task, you should add a possibility to input multi-line text. Please add this.